test/runtests.jl

```diff
@@ -23,6 +22,10 @@ end
 
 include("TestFunctions.jl")
 
+for target in readdir("util")
+    include("util/$target")
+end
+
```

Comment:
I agree that perhaps some re-org needs to be done. I think I did this as a quick and dirty workaround, so I could see the test outputs before the BigBED bug came along threw errors and stopped the testing.